### PR TITLE
Add Fiber.Var benchmarks

### PR DESCRIPTION
**Pull request by jeremiedimino**
And remove duplicated ivar benchmark

### CODE REVIEW

File: bench/micro/fiber_bench.ml [hunk 1]
```diff
@@ -27,21 +27,29 @@ let%bench_fun ("ivar" [@indexed jobs = [ 100; 10_000; 1_000_000 ]]) =
...
-      [ Fiber.Fill (!ivar, ()) ])
-    (let rec loop = function
-       | 0 -> Fiber.return ()
-       | n ->
-         ivar := Fiber.Ivar.create ();
-         let* () = Fiber.Ivar.read !ivar in
-         loop (n - 1)
-     in
-     loop jobs)
+let%bench_fun ("Var.set" [@indexed jobs = [ 100; 10_000; 1_000_000 ]]) =
```
Comment (rgrinberg) [inline bench/micro/fiber_bench.ml:30]: For var, I think it's enough to jobs = 1000 (or whatever value). Probably same for the ivar benchmarks as well. I don't see any non linearity arising here.

**Review by rgrinberg — APPROVED** [review]

File: bench/micro/fiber_bench.ml (hunk 1 above)
Comment (jeremiedimino) [reply bench/micro/fiber_bench.ml:30]: Sounds about right. In fact, for `bind` as well.

Comment (rgrinberg) [reply bench/micro/fiber_bench.ml]: For bind, I just want to make sure we aren't leaking memory by waiting until the last bind to release everything. I'm pretty sure fiber doesn't suffer from this problem though :)

Comment (jeremiedimino) [reply bench/micro/fiber_bench.ml]: I did that

Comment (jeremiedimino) [reply bench/micro/fiber_bench.ml]: Ah, I see. We have tests for this kind of stuff in the fiber expect tests,  so we could add some more tests for this.

**Pull request merged**